internal/policy/policy.go

```diff
@@ -669,6 +674,9 @@ func ListRules(ctx context.Context, repo *git.Repository) ([]*DelegationWithDept
 
 	topLevelTargetsMetadata, err := state.GetTargetsMetadata(TargetsRoleName)
 	if err != nil {
+		if errors.Is(err, ErrMetadataNotFound) {
```

Comment:
could you instead add a `state.HasRole` check before this? It's easier to read / reason through what's happening.

Reply:
Fixed